libs/movex-core-util/src/lib/core-types.ts

```diff
@@ -28,7 +29,9 @@ export namespace NestedObjectUtil {
     20,
     ...0[]
   ];
-
+  
+  export type EmptyObject = Record<string, never>;
```

Comment:
Hey @Abhishek-Mallick , this should be outside of the `NestedObjectUtil` namespace as it cannot be accessed from otherwise and creates compiling issues. See the Github Action checks! 🙌